config/index.js

```diff
@@ -75,57 +160,125 @@ module.exports = {
     'no-obj-calls': 'error',
     'no-octal': 'error',
     'no-octal-escape': 'error',
+    'no-param-reassign': 'error',
     'no-path-concat': 'error',
+    'no-plusplus': 'error',
+    'no-process-env': 'off',
+    'no-process-exit': 'error',
```

Comment:
Not sure about this one 🤔. This rule makes sense for the extension, but not for our Node.js scripts.

Reply:
I think it's good for a default—its usage should be sparse enough that client projects can override it when needed/disable it for their projects.

Reply:
OK, fair enough